cli/script/command-parser.ts

```diff
@@ -273,7 +273,10 @@ var argv = yargs.usage(USAGE_PREFIX + " <command>")
             .demand(/*count*/ 1, /*max*/ 2)  // Require one non-optional and one optional argument.
             .example("login", "Logs in to the CodePush server")
             .example("login --accessKey mykey", "Logs in on behalf of the user who owns and created the access key \"mykey\"")
-            .option("accessKey", { alias: "key", default: null, demand: false, description: " Access key to authenticate against the CodePush server with, instead of providing your username and password credentials", type: "string" })
+            .example("login --proxy http://someproxy.com:455", "Logs in with the specified proxy url")
+            .option("accessKey", { alias: "key", default: null, demand: false, description: "Access key to authenticate against the CodePush server with, instead of providing your username and password credentials", type: "string" })
+            .option("proxy", { default: null, demand: false, description: "URL of the proxy server to use", type: "string" })
+            .option("noProxy", { default: false, demand: false, description: "Bypass the system-wide proxy settings", type: "boolean" })
```

Comment:
`proxy` and `noProxy` sound mutually exclusive.  Globally rename `noProxy` to `bypassProxy`.